auditors/netpols/fixtures/namespace-allow-missing-default-deny-ingress-old-label.yml

```diff
@@ -0,0 +1,19 @@
+# this is to test backwards compatibility with old unregistered annotations (kubernetes.io)
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace-allow-missing-default-deny-ingress-old-label
+  labels:
+    audit.kubeaudit.io/namespace.allow-non-default-deny-ingress-network-policy: "SomeReason"
```

Comment:
question: Is this supposed to be `audit.kubernetes.io/namespace.allow-non-default-deny-ingress-network-policy: "SomeReason"`?

Reply:
great catch! yes! thank you!